hack/lib/build/binaries.sh

```diff
@@ -219,6 +219,17 @@ os::build::internal::build_binaries() {
         local_ldflags+=" -s"
       fi
 
+      #Add Windows File Properties/Version Info and Icon Resource for oc.exe
+      if [[ $platform == "windows/amd64" ]]; then
+        os::build::version::get_vars
+        go get -u github.com/josephspurrier/goversioninfo
```

Comment:
In the script here we should just

```
os::util::ensure::gopath_binary_exists 'goversioninfo'
```

Then we install the dep in the environment doing the build first, not as part of this process. We can add a feature to `os::util::ensure::gopath_binary_exists` that will `go install` if it is not present maybe